frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/components/utils.tsx

```diff
@@ -19,3 +22,26 @@ export const getIconPropsForSource = (
   const iconProps = iconModel ? getIcon({ model: iconModel }) : undefined;
   return iconProps;
 };
+
+export const getJoinedTablesWithIcons = (question: Question) => {
```

Comment:
you might someone else to double check you on this part...

Reply:
Alex P helped me with this part